### ISSUE TITLE
JsonLdOptions: compactArrays

### ISSUE THREAD

**Issue by tobiasschweizer** [digitalbazaar/jsonld.js#195]
May I suggest to add the information to the README that there is an optional argument for `compact`:  JsonLdOptions <https://github.com/digitalbazaar/jsonld.js/blob/21d5290bf023d090f3dd9d0c57e4495dd34465e5/js/jsonld.js#L65>, <https://www.w3.org/TR/json-ld-api/#idl-def-JsonLdOptions>?

This may be done for other methods, too.

The Spec defines compactArrays as follows:
> If set to true, the JSON-LD processor replaces arrays with just one element with that element during compaction. If set to false, all arrays will remain arrays even if they have just one element.

<https://www.w3.org/TR/json-ld-api/#widl-JsonLdOptions-compactArrays>

I made the observation that `compactArrays: false` leads to the effect that everything becomes an array (even if it was not in the JSON that was originally submitted and passed to the JSON-LD processor). Is that the desired behaviour (<https://github.com/jsonld-java/jsonld-java> does the same)?

**Comment by tobiasschweizer** [comment]
As for the README: I could add the information myself. Shall I fork your repo and make a pull request?

**Comment by rmeissn** [comment]
I don't think it is the desired behaviour to convert everything to an array. I stated this in #247 .
